docs/slides/content/social-contracts.md

```diff
@@ -128,21 +129,22 @@ If you will work in multiple teams think about what are the aspects of your Soci
 
 
 #### Facilitation Tips
-Ensure things in the Social Contract are actionable. For example  "Be Open" has good intent but lacks specificity. Ask questions to tease our greater meanings.
+Ensure that things in the Social Contract are actionable. For example  "Be Open" has good intent, but lacks specificity. Ask questions to tease our greater meanings.
 </br>
 
-_"Can you give me an example of not being open?" could lead to actionable items like "Listen to others as I would like to be heard" or "Give space to the quiet person"._ <!--{.element: style="font-size: ; font-weight: 300!important;"} -->
+_"Can you give me an example of not being open?" can lead to actionable items like "Listen to others as I would like to be heard" or "Give space to the quiet person"._ <!--{.element: style="font-size: ; font-weight: 300!important;"} -->
 
 
 
 #### Example - WHO <!-- .element: class="title-bottom-left" -->
 <!-- .slide: data-background-image="images/social-contracts/real-life-example-2.png", data-background-size="contain", class="white-style" -->
 
-
+<!-- Tech Editor: I don't know what "WHO" stands for in this title. You've used World Health Organization before. If you intend to use it again, I'd write it out. -->
+<!-- Tech Editor: Similarly, I don't know what "used practices" means in the next title. Used by whom, for what? I'd just say "Related Practices" as the title. -->
 
 <!-- .slide: data-background-image="images/chef-background.png", data- class="white-style" -->
 ### Related & Used Practices
```

Comment:
Related and Used in this deck or activity

Reply:
Ok. I think "Related" covers it, but if "Related & Used" is what you have used throughout and plan to stick with it, I will also ignore this change going forward.